node/src/BaseClient.ts

```diff
@@ -860,6 +861,29 @@ export class BaseClient {
         return this.createWritePromise(createSismember(key, member));
     }
 
+    /** Removes and returns one random member from the set value store at `key`.
+     * See https://redis.io/commands/spop/ for details.
```

Comment:
Please refer to the spopCount function if the user wants to pop multiple members